src/style.css

```diff
@@ -1 +1,71 @@
-/* Styles go here */
+body {
+  margin: 0;
+  --accent_colour: #00acdc;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
```

Comment:
default value, can be removed

Reply:
I don't quite understand what you mean. If you remove these 4 lines, the font will change to the standard one

Reply:
i meant only last one, sorry for confusing

Reply:
still not fixed